template.tpl

```diff
@@ -36,6 +36,22 @@ ___TEMPLATE_PARAMETERS___
     "simpleValueType": true,
     "help": "If enabled, the server only accessible FPID cookie, generated by UA/GA4 client, will be duplicated to FPIDP cookie, which will be accessible from the client JS. Highly recommend using this option only in case it is necessary."
   },
+  {
```

Comment:
@vriken now here is duplicate.

Reply:
Getting real messy with my commits :s. Sorry about that

Reply:
Should be fixed now